doc/source/dev/database_session_management.md

```diff
@@ -0,0 +1,34 @@
+# How SQLAlchemy sessions are handled in Galaxy
+
+## Access to `sessionmaker`
+
+We have a central resource from which to grab SQLAlchemy's [sessionmaker](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.sessionmaker), that is `app.model.session`.
```

Comment:
Although this is not incorrect (i.e., sessionmaker is the factory that produces a new Session object), it's not the resource we use from our code. We use a Session object registry (i.e., scoped session), which only calls upon the sessionmaker when it needs a new Session object produced.